cmd/updater/main.go

```diff
@@ -828,7 +845,11 @@ func ReadBuilds(parent context.Context, group configpb.TestGroup, builds Builds,
 		appendColumn(grid, heads, nameCfg, rows, *c)
 		alertRows(grid.Columns, grid.Rows, failsOpen, passesClose)
 		if c.Started < stop.Unix() { // There may be concurrency results < stop.Unix()
-			log.Printf("  %s#%s before %s, stopping...", group.Name, c.ID, stop)
+			logrus.WithFields(logrus.Fields{
+				"group": group.Name,
+				"id":    c.ID,
+				"stop":  stop,
+			}).Info("Group started before latest time, stopping")
```

Comment:
What case is this handling, out of curiosity?

Reply:
Done

stops processing earlier columns.